src/components/InfoTile/info-tile.scss

```diff
@@ -0,0 +1,118 @@
+//===================
+//SMALL TILE STYLES
+//===================
+.ibm--col-lg-4.tile--sm {
+  max-width: rem(288px);
+}
+
+.tile--sm {
+  display: flex;
+  flex-direction: column;
+}
+
+.title--sm {
+  font-size: rem(28px);
```

Comment:
We can use

 `@include type-style('productive-heading-04');`

for this. You can see the type tokens here https://next.carbondesignsystem.com/guidelines/typography/productive

Reply:
and if we want to use the fluid type we have to use the class name (there is a conflict with the breakpoints from carbon and elements right now)

`className="bx--type-expressive-heading-04"` would give us the fluid version of that type token. (although not sure if that's what design wants 🤷‍♂️ )